requests_toolbelt/adapters/appengine.py

```diff
@@ -44,6 +49,57 @@ def __init__(self, validate_certificate=True, *args, **kwargs):
         super(AppEngineAdapter, self).__init__(self, *args, **kwargs)
 
     def init_poolmanager(self, connections, maxsize, block=False):
-        self.poolmanager = gaecontrib.AppEngineManager(
-            validate_certificate=self._validate_certificate
-        )
+        self.poolmanager = _AEPoolManager(self._validate_certificate)
+
+
+class _AEPoolManager(object):
```

Comment:
Nit: Why not `_AppEnginePoolManager`? I'm not a fan of abbreviations like this. (but this is also not my project. ;))